python/ray/train/v2/tests/test_v2_api.py

```diff
@@ -1,11 +1,13 @@
 import importlib
+import pickle
```

Comment:
ray uses `ray.cloudpickle` instead:

```suggestion
import ray.cloudpickle as ray_pickle
```